R/fb_plot_number_species_by_trait.R

```diff
@@ -0,0 +1,82 @@
+#' Show Number of Species per Trait
+#'
+#' @inheritParams fb_get_coverage
+#'
+#' @param threshold_species_number `NULL` or `numeric(1)` \[default = `NULL`\]\cr{}
+#'        If `NULL` doesn't show a line otherwise adds a vertical line
+#'        at the specified number of species
+#'
+#' @return a ggplot2 object
+#'
+#' @examples
+#' data(species_traits)
+#' 
+#' \dontrun{%
+#' fb_plot_number_species_by_trait(species_traits)
+#' 
+#' # Add a vertical cutoff line
+#' fb_plot_number_species_by_trait(species_traits, 100)
```

Comment:
Wouldn't it be better to stick with a percentage as in the functions `fb_filter_*()`?

Reply:
100% right :) I forgot to change the name of the thresholds and their nature so that everything is in line.